shared/middleware/files_api.rb

```diff
@@ -224,10 +224,12 @@ def get_file(endpoint, encrypted_channel_id, filename, code_projects_domain_root
     not_found if result[:status] == 'NOT_FOUND'
     not_modified if result[:status] == 'NOT_MODIFIED'
     last_modified result[:last_modified]
-
+    # abuse_threshold here should match ABUSE_THRESHOLD in project.js to keep
+    # our abuse score threshold consistent for projects and their assets.
+    abuse_threshold = 15
```

Comment:
there is a way to share constants programmatically. Please add this constant to shared_constants.rb, and then run `cd apps ; grunt exec:generateSharedConstants`. if that doesn't work try 
```
`npm bin`/grunt exec:generateSharedConstants
```
this will make them show up in apps/src/util/sharedConstants.js .